src/walnut.ts

```diff
@@ -255,6 +260,26 @@ const hallways: Hallway[] = [
     new Room("2218"),
   ]),
 
+  // hallway 2401
+  new Hallway([
+    new Fork(BACK, ConnectionNode.C2401_T0_2100S, "the 2100s"),
+    new Room("2401", LEFT, {
+      prefix: "the",
+      aliases: ["2401", "Athletic Director's Office"]
```

Comment:
Aliases are alternate names for a room. 2401 is already the name of the room, so you don't need it as an alias.
```suggestion
      aliases: ["Athletic Director's Office"],
```